### DISCUSSION TITLE
Pequeñas dudas en práctica 1

### DISCUSSION THREAD

**Discussion by maxkario** [Launch-X-Latam/MisionFrontEnd#62] [Q&A] [answered]
Saludos, me surgió una duda analizando los requerimientos.
Cuando solicitan un dashboard sobre el dinero recibido, no están especificando un tiempo determinado. Es decir,  que es lo que se quiere: el dinero recibido diario, en la semana o al mes? Para poder saber que cantidad se recibió se necesita conocer un rango de tiempo específico.
Por otra parte, y es algo más chistoso para mi, porqué el cliente elige los colores, si solo quiere que su demanda sea exitosa? 🤣

**Comment by omarfrias-maker** [comment]
Muchas veces elige los colores por identidad de la marca o respetando la paleta del logotipo.

**Comment by OscarHB1** [comment]
Hola

Yo entiendo que por "cliente" se refiere al despacho de abogados que solicita que se desarrolle una página web.
Digamos nosotros somo los desarrolladores a los que el despacho acude para hacer la página web, por lo tanto el despacho de abogados son nuestros clientes.

Espero haberte ayudado

**Comment by OATS74** [comment]
Están un poco adelantado o es idea mía. Para cuando hay que entregar ese proyecto

**Reply by renatocoello** [reply]
Realmente creo que si va un poco adelantado
Hay que ir despacio, no vale la pena adelantarse

**Reply by mikaelzen** [reply]
Es la práctica del modulo 1,  así que no está muy adelantado, son 7 módulos para revisar en 6 semanas; así que en pormedio vamos a avanzar al menos uno por semana. 
En lo personal, me ha tomado bastante tiempo desarrollar esta práctica, así que si pueden ir avanzando, mejor.
Aunque bueno, lo  mio fue porque me puse a probar como 20 wireframes distintos y eso me tomó más tiempo.

**Reply by maxkario** [reply]
Pienso que realmente es lo que se debería de ir discutiendo ya que como dice el compañero, es parte de la primera practica. Y como digo anteriormente son dudas que surgen al momento de analizar el enunciado. :)

**Comment by JosafatJimenezB** [comment, marked as answer]
bros, para esta semana apenas andamos con la introducción al desarrollo ya que es primordial ir paso a paso, entiendo que todos tenemos la emocion de crear apps lo mas rápido posible pero vamos con calma

para esa practica del abogabot solo tienes que plantear como le darias solucion a ese problema y no hay como tal una solucion correcta
les dejo mi practica del abogabot la cual se puede cambiar por otra [link practica](https://github.com/JosafatJimenezB/MissionFrontEnd/tree/main/01%20-%20Intro/Practica%201)

**Reply by AngelArielGOOD** [reply]
Me fui muy util tu aportacion gracias, lo que no entiendo casi es de la practica del publico objetivo, sabes a que se refiere

**Reply by JosafatJimenezB** [reply]
Es en base a que tienes que estudiar como las personas van a interactuar con tu aplicación y como ayudarías a satisfacer sus necesidades

eso fue lo que entedi

**Reply by maxkario** [reply]
Bueno por eso digo al principio que estaba analizando el enunciado de los requerimientos y me surgieron esas dudas. 
Me gusta tu aporte. Seguiré analizando a ver como se resuelve el problema. El live de hoy me hizo comprender un poco mas sobre el tema en cuestión.
